src/CMakeLists.txt

```diff
@@ -683,7 +665,8 @@ if (WITH_CEPHFS)
     ceph_mds.cc)
   add_executable(ceph-mds ${ceph_mds_srcs})
   target_link_libraries(ceph-mds mds ${CMAKE_DL_LIBS} global-static ceph-common
-    Boost::thread)
+    Boost::thread
+    ${ALLOC_LIBS})
```

Comment:
I guess this is even more than we had in the past but actually unifying the allocator is a desired thing IMHO.